### PR TITLE
Issue/603 forecast client 2

### LINKED ISSUE #603: Add module for forecast Ad app
Similar to  like so openclimatefix/ocf-infrastructure@c4eea72 #603

Need to add app for running Ad forecast

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

Add ecs task for forecast ad

Fixes #603 

its probably better to merge #602 first, as its smaller

## How Has This Been Tested?

CI tests

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: terraform/india/development/main.tf [hunk 1]
```diff
@@ -94,8 +96,18 @@ resource "aws_secretsmanager_secret" "satellite_consumer_secret" {
   name = "${local.environment}/data/satellite-consumer"
 }
 
+# 3.1
```
Comment (Sukh-P) [inline terraform/india/development/main.tf:99]: This is meant to be 3.2 I think?

Comment (peterdudfield) [reply terraform/india/development/main.tf]: yea, thanks

File: terraform/india/development/main.tf [hunk 2]
```diff
@@ -296,7 +308,61 @@ module "forecast" {
...
+    { "name" : "AWS_REGION", "value" : var.region },
+    { "name" : "ENVIRONMENT", "value" : local.environment },
+    { "name" : "LOGLEVEL", "value" : "DEBUG" },
+    { "name" : "NWP_ECMWF_ZARR_PATH", "value": "s3://${module.s3-nwp-bucket.bucket_id}/ecmwf/data/latest.zarr" },
+    { "name" : "NWP_GFS_ZARR_PATH", "value": "s3://${module.s3-nwp-bucket.bucket_id}/gfs/data/latest.zarr" },
+    { "name" : "SATELLITE_ZARR_PATH", "value": "s3://${module.s3-satellite-bucket.bucket_id}/data/latest/iodc_latest.zarr.zip" },
+    { "name" : "SENTRY_DSN",  "value": var.sentry_dsn},
+    { "name" : "USE_SATELLITE", "value": "True"}
+    # TODO something about Client name
+      ]
```
[the remark this answers is not in the document]
Comment (peterdudfield) [reply terraform/india/development/main.tf:348]: thanks

**Review by Sukh-P — APPROVED** [review]
Thanks for doing this!

**Pull request merged**